src/app/campaign/store/reducers/flight.reducer.ts

```diff
@@ -115,6 +118,12 @@ export function reducer(state = initialState, action: CampaignActions): State {
         state
       );
     }
+    case ActionTypes.CAMPAIGN_SAVE: {
```

Comment:
This was just a bug I noticed where Save wasn't handling temporary flights that we set to softDeleted.